src/components/Dragons/Dragons.jsx

```diff
@@ -0,0 +1,44 @@
+import { Container } from '@material-ui/core';
+import React, { useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useSelector, useDispatch } from 'react-redux';
+import DragonWidget from './DragonWidget';
+import { getDragons } from '../../redux/dragons/dragons';
+
+const useStyles = makeStyles({
+  root: {
+    padding: '1rem',
+  },
+});
+
+const Dragons = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getDragons());
+  });
```

Comment:
useEffect(() => {
    dispatch(getDragons());
  }, []); <- you can add this Empty [] as parameter to useEffect this means that it only run once the component is mounted, otherwise this will run at any change of state